operator/src/simulation/spec.rs

```diff
@@ -34,6 +34,11 @@ pub struct SimulationSpec {
     /// Enable dev mode for the simulation. This will remove resource limits that are not
     /// explicitly set in the simulation spec.
     pub(crate) dev_mode: Option<bool>,
+    /// Log level to use. On the manager writes to stdout, and on the workers will write
+    /// files for the simulation. The files may not be flushed until the simulation is complete
+    /// which consumes RAM, and will disappear if there is no persistent volume when the pod exits.
```

Comment:
More of an exploratory question: Can we define constraints on memory while running a simulation? In the resource definitions of the managers/workers pods I did not notice and resource constraint fields. How do we define what resources to give to a manger/worker during a simulation run